packages/eventual-send/package.json

```diff
@@ -13,7 +13,7 @@
     "lint-fix": "yarn lint:eslint --fix && yarn lint:types",
     "lint-check": "yarn lint",
     "lint": "yarn lint:types && yarn lint:eslint",
-    "lint:types": "tsc -p jsconfig.test.json",
```

Comment:
Yes. Though I wonder whether we can further simplify… if the one tsconfig always emits, what would break?

Reply:
> Yes. Though I wonder whether we can further simplify… if the one tsconfig always emits, what would break?

Please no.

I'm willing to have a build step before publishing the package, but introducing one that can get out-of-sync with your changes to the sources is horrible.  IIUC, the language server will use the `*.d.ts` for types if they exist, even if they are not up-to-date wrt the `*.js` files they describe.

Reply:
> introducing one that can get out-of-sync with your changes to the sources is horrible

Agreed.

> IIUC, the language server will use the *.d.ts for types if they exist, even if they are not up-to-date wrt the *.js files they describe.

We're in agreement here too. My statement was qualified by "if one tsconfig always emits", by which I meant even the language server, but you're right that [only the TS compiler emits](https://www.typescriptlang.org/docs/handbook/declaration-files/dts-from-js.html#run-the-compiler).